packages/eas-cli/src/commands/update/index.ts

```diff
@@ -120,6 +122,12 @@ export default class UpdatePublish extends EasCommand {
       description: `Emit "eas-update-metadata.json" in the bundle folder with detailed information about the generated updates`,
       default: false,
     }),
+    'rollout-percentage': Flags.integer({
+      description: `Percentage of users this update should be immediately available to. Users not in the rollout will be served the previous latest update on the branch, even if that update is itself being rolled out. The specified number must be an integer between 1 and 100. When not specified, this defaults to 100.`,
+      required: false,
```

Comment:
maybe we can hide the flag from the help for now, so we can clean up the UI to differentiate between the old rollouts and this type of new rollout. 

```suggestion
      hidden: true,
      required: false,
```